src/System.ValueTuple/ref/System.ValueTuple.csproj

```diff
@@ -39,7 +37,7 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp' Or '$(TargetGroup)'=='uap'">
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp' Or '$(TargetGroup)'=='uap' Or '$(TargetGroup)'=='netfx'">
```

Comment:
For netfx it shouldn't be using System.Runtime. It should be the same references as the portable one above. 

You can also remove the netstandard1.0 item group.